emmet-core/emmet/core/classical_md.py

```diff
@@ -0,0 +1,74 @@
+"""Schemas for classical MD package."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Optional
+
+from monty.json import MSONable
+
+from emmet.core.vasp.task_valid import TaskState
+from pydantic import BaseModel, Field
+
+
+@dataclass
+class MoleculeSpec(MSONable):
+    """A molecule schema to be output by OpenMMGenerators."""
+
+    name: str
+    count: int
+    formal_charge: int
+    charge_method: str
+    openff_mol: str  # a tk.Molecule object serialized with to_json
+
+    # @field_validator("openff_mol")
+    # @classmethod
+    # def _validate_openff_mol(cls, v: str) -> str:
+    #     try:
+    #         tk.Molecule.from_json(v)
+    #     except Exception as e:
+    #         raise ValueError(
+    #             "MoleculeSpec.openff_mol must be able to be"
+    #             "parsed with Molecule.from_json."
+    #         ) from e
+    #     return v
+
+
+class ClassicalMDTaskDocument(BaseModel, extra="allow"):  # type: ignore[call-arg]
+    """Definition of the OpenMM task document."""
+
+    tags: Optional[list[str]] = Field(
+        [], title="tag", description="Metadata tagged to a given task."
+    )
+    dir_name: Optional[str] = Field(
+        None, description="The directory for this VASP task"
+    )
+    state: Optional[TaskState] = Field(None, description="State of this calculation")
+
+    calcs_reversed: Optional[list] = Field(
+        None,
+        title="Calcs reversed data",
+        description="Detailed data for each VASP calculation contributing to "
+        "the task document.",
+    )
+
+    interchange: Optional[str] = Field(
+        None, description="Final output structure from the task"
```

Comment:
I think it would make more sense to call this `structure` or `trajectory` or `system` or something along those lines (not everyone will be familiar with what an "interchange" is . Or if nothing else, perhaps include as an alias?